Makefile

```diff
@@ -56,3 +52,19 @@ audit: ## Run audit against real CAI dump data
 
 help: ## Prints help for targets with comments
 	@grep -E '^[a-zA-Z._-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "make \033[36m%- 30s\033[0m %s\n", $$1, $$2}'
+
+
+# Cloudbuild doesn't allow us to use the Dockerfile "ARG" feature so we have to
```

Comment:
This seems rather hacky. Could we instead add a cloudbuild.yaml in the /cloudbuild/ directory which would invoke the Docker builder and properly pass through arguments? Like this example here - https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values#using_default_substitutions

Reply:
Unfortunately, it doesn't support substitutions for dockerfiles (as far as I can tell).

Reply:
Specifically, I'd suggest we leave arguments in the Dockerfile then add a cloudbuild.yaml which does something like this:

````
# Uses the docker build step to build an image called my-image
- name: 'gcr.io/cloud-builders/docker'
  args: ['build', '-t', 'gcr.io/$PROJECT_ID/my-image:$_REGO_VERSION', '., '--build-arg', 'REGO_VERSION=$_REGO_VERSION]
```

Reply:
I attempted to convert the change to this format.  Cloudbuild local worked, but when sending to GCP, it wasn't working, so I think this will have to be a future cleanup.

Reply:
Fair enough, I think that's reasonable.